docs/new/posts/Percona Monitoring and Management/pmm-3.1.0-release.md

```diff
@@ -0,0 +1,26 @@
+---
+date: 2025-03-31
+description: >
+  Percona Monitoring and Management 3.1.0 has been released on Monday, March 31, 2025.
+authors: [catalinaadam]
+categories:
+  - Percona Monitoring and Management
+tags:
+  - Percona Monitoring and Management
+---
+
+# Percona Monitoring and Management 3.1.0 has been released
+
+<!-- more -->
+
+We're excited to announce the release of [Percona Monitoring and Management](https://docs.percona.com/percona-monitoring-and-management/3/index.html){:target="_blank"} 3.1.0, available since Monday, March 31, 2025. This release enhances MongoDB monitoring and refines role-based access controls, along with various usability improvements and bug fixes.
+
+Get hands-on with PMM 3.1.0 by setting up your instance using our [quickstart guide](https://docs.percona.com/percona-monitoring-and-management/3/quickstart.html){:target="_blank"}.
```

Comment:
[quickstart guide](https://docs.percona.com/percona-monitoring-and-management/3/quickstart.html) - This link leads to 404